packages/@react-spectrum/listbox/test/ListBox.test.js

```diff
@@ -893,12 +900,7 @@ describe('ListBox', function () {
       let listbox = getByRole('listbox');
       let options = within(listbox).getAllByRole('option');
       expect(options.length).toBe(5);
-      let isReact19 = parseInt(React.version, 10) >= 19;
```

Comment:
These test are now consistent React 19 which is nice

Reply:
The load more scroll logic now uses the scrollable body's client height vs the scroll height rather to check if it should load more rather than being dependent on us mocking the height of the items and having ScrollView's calculations rect updates happening at the right times. This made it more straight forward to mock.

However, there is still some weird behavior when I just tried to get rid of useVirtualizer and rely on attaching the scroll handler on the ref since some of these tests relied on onScroll being called even with a scroll event doesn't actually trigger via the test (gets called by ScrollView). I think the core issue there is that these tests aren't very representative of a real flow (isLoading doesn't get set/updated by onLoadMore and the scrollHeight isn't updated when onLoadMore is finished being called so it isn't actually simulating additional items being loaded) hence I saw extra load more calls happening since a "load" would finish but then the height wouldn't be changed and thus it would try to load again. Will try to resolve after break, was some additional weirdness that I couldn't quite puzzle out.